software/downloads.md

```
@@ -0,0 +1,19 @@
+---
+
+Head over to the [ModelFLOWs-app](https://github.com/modelflows/ModelFLOWs-app) to access all available resources.
+
+The following ModelFLOWs-app versions are available:
+* Desktop version (v 0.2): [Download](https://github.com/modelflows/ModelFLOWs-app/archive/refs/heads/desktop-version-v0.2.zip)
+* Desktop version (v 0.1): [Download](https://github.com/modelflows/ModelFLOWs-app/archive/refs/heads/desktop-version-v0.1.zip)
+* Web-browser version (v 0.1): [Download](https://github.com/modelflows/ModelFLOWs-app/archive/refs/heads/web-browser-version.zip)
+
+**Note: For the web-browser version, please download the following [databases](https://drive.google.com/drive/u/3/folders/1v-BU5kT8arbDrgGndZir-Et0K4MIrDs1) and insert them in the web-browser version folder.**
+
+Download the installation and set-up tutorial clicking [here](https://github.com/modelflows/ModelFLOWs-app/blob/main/v0.1_Tutorial_ModelFLOWs-app.pdf)
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)